pkg/util/azureclient/applens/applens_headers.go

```diff
@@ -0,0 +1,12 @@
+package applens
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+const (
+	headerXmsClientRequestId string = "x-ms-client-request-id"
+	headerXmsDate            string = "x-ms-date"
+	headerXmsPathQuery       string = "x-ms-path-query"
+	headerXmsRequestId       string = "x-ms-request-id"
+	headerXmsVerb            string = "x-ms-verb"
```

Comment:
> So I updated the client to always suffix "./default" to the end of the scope

After fixing the scope, I received this error:

![image](https://user-images.githubusercontent.com/69012315/180829152-e53e62db-e4fb-486e-9a54-b1ac428576f2.png)

So I added `the x-ms-verb` header and included it in the request the same way all other headers are added.